docs/marvel-query.md

```
@@ -4,33 +4,33 @@
 
 ## Initialization and Query Creation
 
-The init() function is responsible for setting up the library with your API keys and any custom configurations you need for handling requests. Once initialized, it returns a query() function, which you’ll use to create queries for different Marvel API endpoints.
+The init() function is responsible for setting up the library with your API keys and any custom configurations you need to handle requests. Once initialized, it returns a query() function, which you’ll use to create queries for different Marvel API endpoints.
 
-For a detailed explanation of available configuration options, such as global parameters, custom fetch functions, and error handling, refer to the [Configuration Options](configuration.md) page.
+The [Configuration Options](configuration.md) page provides a detailed explanation of available configuration options, such as global parameters, custom fetch functions, and error handling.
 
 | Function                                         | Arguments                                                | Returns                         | Description                                                  |
 | ------------------------------------------------ | -------------------------------------------------------- | ------------------------------- | ------------------------------------------------------------ |
 | `init`                                           | [`keys: APIKeys`](#apikeys), [`config: Config`](#config) | [`query()`](#createquery)† | Initialize the library with public and private keys, and options such as global parameters and custom functions for requests, results, and http client. |
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)